scripts/pyxivo_tracker_only.py

```diff
@@ -79,6 +79,19 @@ def main(args):
 
     data.sort(key=lambda tup: tup[0])
 
+    # Open the file and remove all comments 
+    json_file = open(args.cfg)
+    json_string = ''.join(re.sub('//.*','',line) for line in json_file)
+        
+    # Parse as json object
+    json_data = json.loads(json_string)
+
+    # Store the descriptor type
+    descriptor_type = json_data['tracker_cfg']['descriptor']
+    
+    # Determine the format for writing descriptor values based on descriptor type
+    descriptor_format = "%f" if descriptor_type in ["SIFT", "SURF"] else "%d"
```

Comment:
I am not completely sure if SIFT and SURF are the only floating point descriptors we support. Also, I am not completely sure if we need to set the descriptor type in the config file when something like ORB

Reply:
There is a little list of descriptors we support on line 133 of tracker.cpp. BRIEF, BRISK, ORB, and FREAK are all [binary descriptors](https://gilscvblog.com/2013/08/26/tutorial-on-binary-descriptors-part-1/). The only remaining ones are SIFT and SURF.